cli/src/main/java/org/infinispan/cli/commands/rest/Create.java

```diff
@@ -54,15 +56,18 @@ public CommandResult exec(ContextAwareCommandInvocation invocation) {
    @CommandDefinition(name = "cache", description = "Create a cache", activator = ConnectionActivator.class)
    public static class Cache extends RestCliCommand {
 
-      @Argument(required = true)
-      String name;
+      @Arguments(required = true)
+      List<String> args;
 
       @Option(completer = CacheConfigurationCompleter.class, shortName = 't')
       String template;
 
       @Option(completer = FileOptionCompleter.class, shortName = 'f')
       Resource file;
 
+      @Option(shortName = 'c')
+      String configuration;
```

Comment:
Do we include this just to be consistent with `-f` and `-t`?

Reply:
Yes